sdk/src/connections/usb/linux/usb_depth_sensor_linux.cpp

```diff
@@ -485,17 +485,35 @@ aditof::Status UsbDepthSensor::writeAfeRegisters(const uint16_t *address,
     cq.size = MAX_BUF_SIZE;
 
     size_t sampleCnt = 0;
+    uint8_t b = sizeof(uint16_t); // Size (in bytes) of an AFE register
```

Comment:
The name should be more suggestive. Maybe 'registerSize', 'regSize' etc. Also, this can be declared as 'const'. 
In order to be more rigorous I think 'sizeof(data[0])' should be used (but this is just a detail as 'sizeof(uint16_t)' is used in lots of other places, so this change would not make any difference)